src/metrics.go

```diff
@@ -256,10 +257,20 @@ func getAttributeType(v interface{}) metric.SourceType {
 
 }
 
-func getStatus(path string) (resp *http.Response, err error) {
-	netClient := &http.Client{
+func httpClient(args argumentList) *http.Client {
```

Comment:
Shouldn't we pass only the necessary arguments to the method? From my point of view, when I see a method with explicit arguments, I can get an extra bit of information to deduce what it does and what it needs. This could be an interesting topic to debate.

Reply:
You are right. Actually I'm removing the whole argument because it relies in a global variable (BTW another bad practice)

Reply:
Hello Team,

That is a great work! Really appreciate the effort you guys put in place.
Do you mind to update https://github.com/newrelic/nri-nginx/blob/master/nginx-config.yml.sample with the new parameter "validate_certs"?

Regards,